### PR TITLE
Move to Python 3.8

### PR DESCRIPTION
**Pull request by FyodorK**
- Adding definitive version of required packages to setup.py, 
- Moving to python3.8, 
- Updating ReadMe.md file against changes

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -73,8 +73,8 @@ def find_version(*file_paths):
 if python_version[0] < 3:
     print("Only Python > 3.5 is supported")
     sys.exit(0)
-elif  python_version[1] < 5:
-    print("Only Python > 3.5 is supported")
+elif  python_version[1] < 8:
+    print("Only Python > 3.8 is supported")
```
Comment (vshampor) [inline setup.py:77]: This is probably asking the user too much. Let's not explicitly prohibit 3.5-3.7 here.

File: setup.py [hunk 2]
```diff
@@ -41,30 +41,30 @@ def find_version(*file_paths):
...
+                    "opencv-python==4.4.0.46",
+                    "pytest-mock==3.3.1",
+                    "prettytable==2.0.0",
+                    "mdutils==1.3.0",
+                    "yattag==1.14.0",
+                    "jsonschema==3.2.0",
+                    "wheel==0.36.1",
+                    "defusedxml==0.7.0rc1",
+                    "mlflow==1.12.1",
                     "returns==0.14"]
```
Comment (vshampor) [inline setup.py:68]: Separate these changes into another PR.

**Review by vshampor — CHANGES_REQUESTED** [review]

File: setup.py (hunk 1 above)
Comment (FyodorK) [reply setup.py:77]: Should these lines be deleted from script?
